### PR TITLE
Update pip CI dependencies

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

**Review by YooSunYoung — APPROVED** [review]

**Comment by YooSunYoung** [comment]
Is the warning about plotting in the failing tests related to this update...?

**Comment by SimonHeybrock** [comment]
> Is the warning about plotting in the failing tests related to this update...?

Yes, I think we should wait until #3200 is merged.

**Pull request merged**
